torch/_dynamo/variables/user_defined.py

```diff
@@ -89,16 +89,7 @@ def is_forbidden_context_manager(ctx):
     except ImportError:
         pass
 
-    try:
-        from torch.testing._internal.jit_utils import (
-            _AssertRaisesRegexWithHighlightContext,
-        )
-
-        f_ctxs.append(_AssertRaisesRegexWithHighlightContext)
-    except ImportError:
-        pass
-
-    return ctx in f_ctxs
+    return ctx.__qualname__ == "_AssertRaisesRegexWithHighlightContext" or ctx in f_ctxs
```

Comment:
Actually, the better thing to do is to use sys.modules.get: if we already have torch.testing._internal.jit_utils imported, then we can grab _AssertRaisesRegexWithHighlightContext.

See https://github.com/pytorch/pytorch/blob/34127fc6880e25e71c62daf8fc8f909a4efddd2d/torch/_dynamo/variables/dicts.py#L712

Reply:
Unfortunately, `jit_utils` is not available in `sys.modules` at this time:

```python
(Pdb++) sys.modules.get('torch.testing._internal.jit_utils')
(Pdb++) sys.modules.get('torch.testing._internal.jit_utils') is None
True
```